Emby.Server.Implementations/Data/SqliteDisplayPreferencesRepository.cs

```diff
@@ -168,19 +167,15 @@ public DisplayPreferences GetDisplayPreferences(string displayPreferencesId, Gui
 
             var guidId = displayPreferencesId.GetMD5();
 
-            using (var connection = GetConnection(true))
```

Comment:
Why did you remove this argument?

Reply:
`GetConnection` never actually uses the argument, so I removed the argument to make it easier in the future to remove that parameter from the method. I was originally going to do it in this PR, but it would've required me to make changes that would conflict with part 1, so I'm saving it until later.

Reply:
I left the parameter in purpose, it used to determine if a read only connection should be returned or a read-write connection. By leaving it in we can easily add that back in the future.

Reply:
Alright, I'll add it back.